content/300-guides/050-database/850-using-prisma-with-planetscale.mdx

```diff
@@ -69,19 +69,14 @@ If you try to push to a production branch, you will get the [error message](/ref
 
 PlanetScale does not allow foreign keys in its database schema. By default, Prisma uses foreign keys in the underlying database to enforce relations between fields in your Prisma schema. In Prisma versions 3.1.1 and later, you can [emulate relations in Prisma Client with the `prisma` relation mode](/concepts/components/prisma-schema/relations/relation-mode#emulate-relations-in-prisma-with-the-prisma-relation-mode), which avoids the need for foreign keys in the database.
 
-The ability to set the relation mode is currently a [preview feature](/concepts/components/preview-features). To enable this, add it to the `previewFeatures` list in the `generator` block of `schema.prisma`, and then set the `relationMode` field to `"prisma"` in the `datasource` block:
+To enable emulation of relations in Prisma Client, set the `relationMode` field to `"prisma"` in the `datasource` block:
```

Comment:
I think the "needed a preview feature flag from vesion x to y" would fit into the admonition below quite nicely